.github/workflows/manage-github-issue-for-outdated-pods.yml

```diff
@@ -0,0 +1,30 @@
+name: Check outdate pod and create issue
+
+on: 
+  schedule:
+    - cron: '0 0 * * 1'
+  workflow_dispatch:
+
+jobs:
+  check-outdated-pods:
+    runs-on: macos-latest
+    name: Check outdate pods and create issue if it doesn't exist
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check outdated pods and create issue
+        id: check-outdated-pods-and-create-issue
+        uses: rudderlabs/github-action-updated-pods-notifier@main
+        with:
+          outdated-pod-names: "FirebaseAnalytics"
```

Comment:
is thsi pod name correct? Just confirming

Reply:
Yes. 
Actually, GA4 is exactly the same as Firebase, just the name is different. Refer to the podscpec [line](https://github.com/rudderlabs/rudder-integration-ga4-ios/blob/d130e32cbd4c1e54898064a945494f82ee798249/Rudder-GA4.podspec#L27).